src/StoredEventRepository.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Spatie\EventProjector;
+
+use Illuminate\Support\Collection;
+use Spatie\EventProjector\Models\StoredEventData;
+
+interface StoredEventRepository
+{
+    public static function retrieveAll(string $uuid = null, int $startingFrom = null): Collection;
```

Comment:
Is it necessary for all these to be static? Makes it harder for implementations that for example need to store a database connection as a property.

Reply:
Shouldn't the DIC be a good solution to remove static calls?

Reply:
So removing the static calls was actually a pretty small change https://github.com/spatie/laravel-event-projector/pull/183/commits/14eeb4aff312e165f5559406fb6be32b0d4005cf and tests are still working. Makes me wonder why they were static in the first place